apps/site/assets/ts/schedule/components/schedule-finder/TripDetails.tsx

```diff
@@ -91,15 +93,19 @@ export const TripDetails = ({
         </tr>
       </thead>
       <tbody className="schedule-table__subtable-tbody">
-        {tripInfo.times.map((departure, index: number) => (
-          <TripStop
-            departure={departure}
-            index={index}
-            showFare={showFare}
-            routeType={tripInfo.route_type}
-            key={departure.schedule.stop.id}
-          />
-        ))}
+        {tripInfo.times.map((departure, index: number) =>
+          departure.schedule ? (
+            <TripStop
+              departure={departure}
+              index={index}
+              showFare={showFare}
+              routeType={tripInfo.route_type}
+              key={departure.schedule.stop.id}
+            />
+          ) : (
+            errorLoadingTrip
```

Comment:
I assume this is the scenario we've stumbled across elsewhere, where a trip can have a prediction but not a schedule? Is it possible that we could still show trip data in this case, or is it just not there?

Reply:
Yeah, added trips can have a prediction but no schedule, as noted in #351 . It might be possible to show trip data in this case but it would need a refactor of at least the `<TripStop />` component.

Reply:
Ok – I can see how that work would be out-of-scope here, so maybe we can add a new Asana task for "schedule finder should show details of added-in trips"? Unless there already is one somewhere and I haven't seen it.